### PR TITLE
Bscharenberg/mt 207

### PR DESCRIPTION
**Pull request by bryn19**
Adds testing frameworks, pulls in the nypr-player component, and adds jukebox-display and stream-switcher components

### CODE REVIEW

File: app/styles/app.scss [hunk 1]
```diff
@@ -1,14 +1,31 @@
-@import "nypr-ui";
-@import "nypr-ui/brand-header";
+// This is just for Bryn, take out later
```
Comment (kedmundson) [inline app/styles/app.scss]: later is now

File: app/styles/components/_nypr-player-overrides.scss [hunk 2]
```diff
@@ -0,0 +1,50 @@
+@import '../mixins/mq';
+
+$nypr-player-medium-breakpoint               : #{$mq_tablet-portrait};
+$nypr-player-medium-and-up                   : "(min-width: #{$nypr-player-medium-breakpoint})" !default;
+$nypr-player-small-only                      : "(max-width: #{$nypr-player-medium-breakpoint - 1})" !default;
+
+
+// text-crawl overflow styles
+.text-crawl {
```
Comment (kedmundson) [inline app/styles/components/_nypr-player-overrides.scss]: What's the reason for a `_nypr-player-overrides.scss` file at this point?

File: app/styles/app.scss (hunk 1 above)
Comment (bryn19) [reply app/styles/app.scss]: but then you wouldn't have all the cool styles!!

File: app/styles/components/_nypr-player-overrides.scss (hunk 2 above)
Comment (bryn19) [reply app/styles/components/_nypr-player-overrides.scss]: w/o these the sticky player doesn't have text. It's just blank

**Pull request merged**